be/src/exprs/jit/jit_engine.cpp

```diff
@@ -223,9 +219,13 @@ void JITEngine::print_module(const llvm::Module& module) {
     LOG(INFO) << "JIT: Generated IR:\n" << str;
 }
 
-void* JITEngine::lookup_function(const std::string& expr_name) {
+void* JITEngine::lookup_function(const std::string& expr_name, bool mast_exist) {
     auto addr = _jit->lookup(expr_name);
     if (UNLIKELY(!addr || UNLIKELY(addr->isNull()))) {
+        if (!mast_exist) {
+            return nullptr;
+        }
+
```

Comment:
change LOG(ERROR) at Line:233 to ROW_LOG << xxx

Reply:
Fixed.